ansible/roles/prometheus-server/defaults/main.yml

```diff
@@ -0,0 +1,28 @@
+---
+
+# Prometheus
+prometheus_version: 2.20.1
+prometheus_tar_gz: "prometheus-{{ prometheus_version }}.linux-amd64.tar.gz"
+prometheus_tar_gz_url: "https://github.com/prometheus/prometheus/releases/download/v{{ prometheus_version }}/{{ prometheus_tar_gz }}"
+
+prometheus_root_dir: /opt/prometheus
+prometheus_config_dir: /etc/prometheus
+prometheus_server_dir: "{{ prometheus_root_dir }}/prometheus-server"
+prometheus_dist_dir: "{{ prometheus_root_dir }}/dist"
+prometheus_data_dir: /var/opt/prometheus/data
+
+prometheus_user: "prometheus"
+prometheus_group: "prometheus"
+
+# Volume mounts
```

Comment:
As above, the comment here is redundant, as the variable is self-descriptive.

Reply:
Removed.